boofuzz/fuzzable.py

```diff
@@ -1,12 +1,12 @@
+from future.builtins import object
 from future.moves import itertools
 
-from .mutation import Mutation
+from boofuzz.mutation import Mutation
```

Comment:
Can we keep this as `from .mutation import Mutation` for consistency? (I'm not sure if there's any semantic difference)